app/views/welcome/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-expand-lg">
   <div class="container-fluid">
     <a class="navbar-brand" href="/">🏋️‍♂️ SpotMe 🏋️‍♂️</a>
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <button style="border:3px solid black;" class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Good catch here!  I'll have to check out how it looks in production.  In general, I'd suggest preferring to make/update a CSS class in the `.scss` file instead of using `style` in the html ... but maybe that didn't work in this case with the Bootstrap template?

Reply:
Honestly, totally spaced that!  I was just fiddling with it in line to get my bearings, and forgot to move it into the `.scss` file, after I figured out a fix.  Will update accordingly, good catch!!